cpp/tests/transform/row_conversion.cpp

```diff
@@ -833,19 +827,7 @@ TEST_F(RowToColumnTests, SimpleString)
   EXPECT_EQ(new_rows.size(), 1);
   for (auto& row : new_rows) {
     auto new_cols = cudf::convert_from_rows(cudf::lists_column_view(*row), schema);
-
     EXPECT_EQ(row->size(), 5);
-    auto const num_columns = new_cols->num_columns();
-
-    cudf::strings_column_view str_col = new_cols->get_column(1).view();
-    std::vector<thrust::host_vector<int8_t>> col_data;
-    std::vector<thrust::host_vector<cudf::size_type>> offset_data;
-    for (int i = 0; i < num_columns; ++i) {
-      offset_data.emplace_back(
-        std::get<0>(cudf::test::to_host<cudf::size_type>(str_col.offsets())));
-      col_data.emplace_back(std::get<0>(cudf::test::to_host<int8_t>(str_col.chars())));
-    }
-
```

Comment:
This code was not involved in verifying the results.
I removed it because it was calling the deprecated chars() function.